templates/.erb-lint.yml

```diff
@@ -1,20 +1,26 @@
 ---
-glob: "**/app/views/**/*.{html,text,js}{+*,}.erb"
 EnableDefaultLinters: true
+exclude:
+  - '**/rails_admin/*'
+  - "vendor/**/*"
 linters:
+  SpaceInHtmlTag:
+    enabled: false
   Rubocop:
     enabled: true
     rubocop_config:
       inherit_from:
         - .rubocop.yml
-      # Each ruby statement (between ERB tags <% ... %>) is parsed and analyzed independently of each other,
-      # so contextual rules are manually disabled
       Layout/InitialIndentation:
         Enabled: false
+      Layout/LineLength:
+        Enabled: true
       Layout/TrailingEmptyLines:
         Enabled: false
       Layout/TrailingWhitespace:
         Enabled: false
+      Layout/BlockAlignment:
+        Enabled: false
```

Comment:
this has false-positives

Reply:
Could you please provide an example?
This seems to be a pretty important rule:
https://docs.rubocop.org/rubocop/cops_layout.html#layoutblockalignment

Reply:
I cannot write:
![CleanShot 2024-09-04 at 17 22 27@2x](https://github.com/user-attachments/assets/027f2853-2ab3-477e-a7d4-c84945df783c)

I'd have to write:
![CleanShot 2024-09-04 at 17 22 49@2x](https://github.com/user-attachments/assets/596393f0-dcb5-43ca-8b2d-a16d8dd0056a)